### PR TITLE
Add "not contains"

### LINKED ISSUE #925: Add "contains not" comparator to vault search

### PR DESCRIPTION
**Pull request by DarkHorseV1**
Add "not contains" comparator to vault search
Fixes #925

### CODE REVIEW

File: src/main/resources/i18n/messages_ru.properties [hunk 1]
```diff
@@ -576,4 +576,6 @@ mapVault.ladder = Рейтинговые карты
 news.showLadderMaps = Показать рейтинговые карты
 ranked1v1.showMaps = Показать рейтинговые карты
 settings.language = Язык
-settings.notifications.displayNotification = Показывать уведомление
\ No newline at end of file
+settings.notifications.displayNotification = Показывать уведомление
```
Comment (1-alex98) [inline src/main/resources/i18n/messages_ru.properties]: Never do translation here but on poeditor

Comment (1-alex98) [reply src/main/resources/i18n/messages_ru.properties]: Got into the setting in the client and then language and u will find a link to it

File: src/main/java/com/faforever/client/query/SpecificationController.java [hunk 2]
```diff
@@ -340,6 +342,9 @@ private Condition getStringCondition(ComparisonOperator comparisonOperator, Stri
     if (comparisonOperator == RE) {
       return prop.eq("*" + value + "*");
     }
+    if (comparisonOperator == EX) {
+      return prop.ne("*" + value + "*");
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/query/SpecificationController.java:346]: Did you test it?

Comment (DarkHorseV1) [reply src/main/java/com/faforever/client/query/SpecificationController.java:346]: yes. I started the client and tried to search for maps in the storage. It works correctly. All tests is passed

File: src/main/resources/i18n/messages_ru.properties (hunk 1 above)
Comment (DarkHorseV1) [reply src/main/resources/i18n/messages_ru.properties]: but why? I made a correct translation into Russian

Comment (1-alex98) [reply src/main/resources/i18n/messages_ru.properties]: because we have an app for translations and if you dont enter it there it gets overwritten

**Review by 1-alex98 — CHANGES_REQUESTED** [review]
Remove translations

**Comment by DarkHorseV1** [comment]
Added Translation deleted.
Now "not contains" comes after "contains".

**Review by 1-alex98 — APPROVED** [review]

**Pull request merged**